egui_table/src/table.rs

```diff
@@ -639,8 +634,14 @@ impl<'a, 'cb> TableSplitScrollDelegate<'a, 'cb> {
 
         for row_nr in row_range {
             let y_range = Rangef::new(
-                self.header_row_y.last() + self.table.get_row_top_offset(row_nr),
-                self.header_row_y.last() + self.table.get_row_top_offset(row_nr + 1),
+                self.header_row_y.last()
+                    + self
+                        .table
+                        .get_row_top_offset(ui.ctx(), self.table_delegate, row_nr),
+                self.header_row_y.last()
+                    + self
+                        .table
+                        .get_row_top_offset(ui.ctx(), self.table_delegate, row_nr + 1),
```

Comment:
this becomes very hard to read now :/

Reply:
Maybe by cloning the `Context` somewhere I can make a simpler helper.